aioambient/util/climate_utils.py

```diff
@@ -0,0 +1,135 @@
+"""Climate utility functions."""
+from __future__ import annotations
+
+from math import log, sqrt
+from typing import cast
+
+MAGNUS_A = 17.27
+MAGNUS_B = 237.7
+
+
+class ClimateUtils:
+    """Climate utility functions."""
+
+    @staticmethod
+    def convert_celsius_to_fahrenheit(celsius: float) -> float:
+        """Convert celsius to fahrenheit."""
```

Comment:
All new method docstrings need to specify args, return types (as appropriate), and exceptions (as appropriate).

Reply:
Fixed.